rolling-shutter/cmd/gnosiskeyper/gnosiskeyper.go

```diff
@@ -0,0 +1,54 @@
+package gnosiskeyper
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/cmd/shversion"
+	keyper "github.com/shutter-network/rolling-shutter/rolling-shutter/keyperimpl/gnosis"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/keyperimpl/gnosis/database"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/medley/configuration/command"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/medley/db"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/medley/service"
+)
+
+func Cmd() *cobra.Command {
+	builder := command.Build(
+		main,
+		command.CommandName("gnosiskeyper"),
```

Comment:
I would remove this and use the `Configuration.Name()` function to return the command name.
(See other comment)